include/net/nrf_cloud.h

```diff
@@ -375,7 +376,7 @@ int nrf_cloud_sensor_data_send(const struct nrf_cloud_sensor_data *param);
  * @brief Update the device shadow with sensor data.
  *
  * @param[in] param Sensor data; the data pointed to by param->data.ptr
- *                  must be a cJSON object (cJSON *).
+ *                  must be a valid JSON string.
```

Comment:
Is this change intended to be in https://github.com/nrfconnect/sdk-nrf/pull/5280/commits/851fbf9a22736c3a3e5ebbb7ef671b540c01dfd3?